MalaDash/src/maladash/src/components/DrawingPanel.java

```diff
@@ -0,0 +1,28 @@
+
+package maladash;
+
+import java.awt.*;
+import java.awt.image.*;
+import java.io.IOException;
+import java.net.URL;
+import javax.imageio.ImageIO;
+import javax.swing.*;
```

Comment:
How many packets do you use?
If some package didn't used pls delete it.

Reply:
Sadly that panel need URL, ImageIO and BufferedImage object to draw a background, in short it use all of them.